app/Libraries/OrderCheckout.php

```diff
@@ -140,6 +211,26 @@ public static function complete($orderNumber)
         return $order;
     }
 
+    /**
+     * Helper method for failing checkout with just the order number.
```

Comment:
Would something like `::for($orderNumber)` a bit more useful? `OrderCheckout::for('blah')->failCheckout()` etc.